srcs/web/WebhookManager.py

```diff
@@ -92,15 +92,20 @@ def workflow_job(self, payload: WebHook):
             }
         )
 
-        runner_m.factory.cloud_manager.update_vm_metadata(
-            payload.workflow_job.runner_name,
-            dict(
-                status=status["status"],
-                repository=payload.repository.name,
-                workflow=payload.workflow_job.workflow_name,
-                job=payload.workflow_job.name
-            )
-        )
+        if (
+            payload.action != "queued"
+            and payload.workflow_job.conclusion != "skipped"
+        ):
+            if payload.workflow_job.runner_group_name != "GitHub Actions":
```

Comment:
It's not enough, you are going to get event for runners that are:
- Both not managed by the runner-manager and not GitHub hosted.
- Managed by another runner-manager.

Reply:
For "Managed by another runner-manager," it's not an issue as the function is also implemented for other runner-managers. Right? Even if nothing will happen
<img width="502" alt="image" src="https://github.com/scality/runner-manager/assets/44836297/d9e95a8b-55dc-47cf-a037-1076391903ad">

Reply:
I have marked the PR as a draft because what I just added is not working. Just let me know if it is what you had in mind. I just can't figure out why, but sometimes the Redis is initialized and other times it's not. I will investigate while waiting!

Reply:
Alternatively, I can use an `if` statement with `"self-hosted" in payload.workflow_job.labels`, but I have a feeling that it won't work every time since we don't always specify "self-hosted" as a tag for the runners.